inference-engine/src/preprocessing/ie_preprocess_itt_domains.hpp

```diff
@@ -0,0 +1,32 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+/**
+ * @brief Defines openvino domains for tracing
+ * @file ie_preprocess_itt_domains.hpp
+ */
+
+#pragma once
+
+#include <openvino/itt.hpp>
+
+namespace InferenceEngine {
+        namespace itt {
+            namespace domains {
+                OV_ITT_DOMAIN(IEPreproc);
+            }
+        }
```

Comment:
BTW, usually we don't use tabs / 4 spaces inside namespaces

Reply:
fixed